docs/reference/rest_api_drafts_records.md

```diff
@@ -1677,7 +1677,7 @@ Content-Type: application/json
 
 {
   "permission": "edit",
-  "expires_at": "2121-03-25T21:06:29.563235"
+  "expires_at": "2024-12-25"
```

Comment:
Ah can you adjust the description in the table above too like:

```
| `expires_at` | string | body | Date string in YYYY-MM-DD format. When the link expires. |
```
?

Reply:
I would add `expires_at` | string | body | ISO 8601 date format. When the link expires.
and update all others to have either:
- `ISO 8601 date` format
- `ISO 8601 date time` format
and it's clear with the example for each request below each one. WDYT @fenekku ?

Reply:
:100: that'd be great